Library/ViewModels/LiveStreamDiscoveryLiveNowCellViewModel.swift

```diff
@@ -48,6 +50,14 @@ LiveStreamDiscoveryLiveNowCellViewModelInputs, LiveStreamDiscoveryLiveNowCellVie
     self.creatorLabelText = liveStreamEvent
       .map { Strings.Creator_name_is_live_now(creator_name: $0.creator.name) }
 
+    self.numberPeopleWatchingHidden = liveStreamEvent.map { $0.numberPeopleWatching == nil }
+
+    self.numberPeopleWatchingText = liveStreamEvent.map {
+      guard let numberWatching = $0.numberPeopleWatching else { return "0" }
+
+      return Format.wholeNumber(numberWatching)
```

Comment:
could skip the guard and just do `Format.wholeNumber($0. numberPeopleWatching ?? 0)` yah?